### ISSUE TITLE
One master corresponds to two slaves, one for synchronous replication and one for asynchronous, can this be achieved?

### ISSUE THREAD

**Issue by iamqq23ue** [apache/rocketmq#5128]
I saw an article mentioning that this model is going to be adopted in the future, but I understand that it can't be done now

**Comment by lizhimins** [comment]
It can also be called semi synchronous replication. For a three replica quorum write, only two replicas are required to return success. For the delayed replica, it is asynchronous

**Comment by azhsmesos** [comment]
Does RocketMQ currently implement a custom synchronization configuration similar to the Quorum NWR algorithm?

**Comment by RongtongJin** [comment]
Hi @iamqq23ue @azhsmesos, I listed some related documents[1][2][3], similar features have been merged into develop branch, and the first version of RocketMQ 5.0 will be released soon. 

[1] https://github.com/apache/rocketmq/blob/develop/docs/cn/controller/deploy.md
[2] https://github.com/apache/rocketmq/wiki/RIP-44-Support-DLedger-Controller
[3] https://github.com/apache/rocketmq/wiki/RIP-34-Support-quorum-write-and-adaptive-degradation-in-master-slave-architecture

**Comment by iamqq23ue** [comment]
> Hi @iamqq23ue @azhsmesos, I listed some related documents[1][2][3], similar features have been merged into develop branch, and the first version of RocketMQ 5.0 will be released soon.
> 
> [1] https://github.com/apache/rocketmq/blob/develop/docs/cn/controller/deploy.md [2] https://github.com/apache/rocketmq/wiki/RIP-44-Support-DLedger-Controller [3] https://github.com/apache/rocketmq/wiki/RIP-34-Support-quorum-write-and-adaptive-degradation-in-master-slave-architecture

It looks very functional, thanks.But  IMO，I noticed that the following quote maybe wrong . In my test, even if SYNC_MASTER is set, two slaves still only need one slave to return ACK and the other to be asynchronous.version 4.9.1.
In addition, is RIP-34 still unable to set which slave is synchronous and which is asynchronous?

quote:
"However, synchronous replication returns only after each copy is confirmed. On the other hand, in synchronous replication mode, if a slave in the group feigned death, the whole transmission would fail until manual processing."
